### PR TITLE
Fix update additional dependencies script

### PR DESCRIPTION
**Pull request by ilaner**
## Related Issues
fixes:

### CODE REVIEW

File: demisto_sdk/scripts/update_additional_dependencies.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def update_additional_dependencies(
         int: 1 if failed, 0 if succeeded
     """
     try:
-        if "poetry.lock" not in GitUtil()._get_all_changed_files():
+        if Path("poetry.lock") not in GitUtil()._get_all_changed_files():
```
Comment (dorschw) [inline demisto_sdk/scripts/update_additional_dependencies.py:27]: Is this safe, considering relative paths?

Comment (ilaner) [reply demisto_sdk/scripts/update_additional_dependencies.py:27]: This function  _get_all_changed_files returns the relative paths that was changed

Comment (ilaner) [reply demisto_sdk/scripts/update_additional_dependencies.py:27]: relative to the git path, but it's the same

Comment (dorschw) [reply demisto_sdk/scripts/update_additional_dependencies.py:27]: Perhaps `CONTENT_PATH / "poetry.lock"` would be safer

Comment (ilaner) [reply demisto_sdk/scripts/update_additional_dependencies.py:27]: but it will not work

**Review by dorschw — APPROVED** [review]

**Pull request merged**